docs/developer/fee-abstraction.md

```diff
@@ -3,9 +3,9 @@ title: Implementing Fee Abstraction in Wallets
 description: How to allow your wallet users to pay for gas fee using alternate fee currencies
 ---
 
-Celo allows paying gas fees in currency other than the native currency. The tokens that can be used to pay gas fees is controlled via governance and the list of tokens allowed is maintained in [**FeeCurrencyWhitelist.sol**](/contracts/core-contracts).
+Celo allows paying gas fees in currency other than the native currency. The tokens that can be used to pay gas fees is controlled via governance and the list of tokens allowed is maintained in [**FeeCurrencyDirectory.sol**](/contracts/core-contracts).
 
-Alternate fee currency works with EOAs and no paymaster is required!
+Alternate fee currencies work with EOAs and no paymaster is required!
```

Comment:
This looks like a marketing tag line. Either move to the top separated by ---, or remove?

Reply:
(and should retain the s)